### PR TITLE
Arch 150 last menu dropdown off page

### PR DESCRIPTION
**Pull request by bixgomez**
# READY FOR REVIEW

# Summary
- ARCH-150
- ARCH-155
- ARCH-171

# Steps to Test
1. Visit the dev site
1. Analyze the main dropdown menu and confirm that ARCH-150 is resolved
1. Analyze the side menu and confirm that ARCH-171 is resolved
1. Edit the home page content, confirming that there is a new "underline-accent" class, and that applying it to a bit of text has the desired effect (per ARCH-155)

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- [ARCH-150](https://stanfordits.atlassian.net/browse/ARCH-150)
- [ARCH-155](https://stanfordits.atlassian.net/browse/ARCH-155)
- [ARCH-171](https://stanfordits.atlassian.net/browse/ARCH-171)

### CODE REVIEW

File: config/default/editor.editor.basic_html.yml [hunk 1]
```diff
@@ -55,13 +55,13 @@ settings:
             - Redo
             - Maximize
   plugins:
+    stylescombo:
+      styles: "a.decanter-button|Primary Button\r\na.decanter-button--big|Big Primary Button\r\na.decanter-button--secondary|Secondary Button\r\na.decanter-more-link|More Link\r\na.decanter-external-link|External Link\r\np.decanter-font-lead|Lead Font\r\np.decanter-font-splash|Splash Font\r\np.decanter-short-line-length|Short Line Length\r\np.decanter-caption|Caption\r\np.decanter-credits|Credits\r\nspan.underline-accent|Underlined Accent Text\r\ntable.decanter-table--borderless|Borderless Table"
```
Comment (pookmish) [inline config/default/editor.editor.basic_html.yml]: This doesn't apply the correct styling to match the mockups. I suggest changing the `span.underline-accent` to `a.underline-accent`. there should never be underlined text that isn't a link anyways (accessibility problems). see screenshot:
screen shot 2018-07-18 at 9 14 54 am

File: docroot/themes/humsci/su_humsci_theme/scss/components/molecules/hover-menu.scss [hunk 2]
```diff
@@ -142,6 +142,13 @@
                 margin-right: 0;
               }
             }
+            // Last two submenus should hug the right.
+            &:last-of-type,
+            &:nth-last-of-type(2) {
+              ul.decanter-nav-submenu {
+                right: 0;
```
Comment (pookmish) [inline docroot/themes/humsci/su_humsci_theme/scss/components/molecules/hover-menu.scss]: maybe we only do this when the screen is small enough? i'm not sure if theres a pure css way...  might need some javascript

File: config/default/editor.editor.basic_html.yml (hunk 1 above)
Comment (bixgomez) [reply config/default/editor.editor.basic_html.yml]: @pookmish I have made the change, but when I do so I don't see that style in any WYSIWYG dropdowns...

File: docroot/themes/humsci/su_humsci_theme/scss/components/molecules/hover-menu.scss (hunk 2 above)
Comment (bixgomez) [reply docroot/themes/humsci/su_humsci_theme/scss/components/molecules/hover-menu.scss]: @pookmish Yes, as Joe and I were discussing yesterday, there is a small window where we have wrapping of the menu -- at that point, those last two get stranded to the right.  So, in the next sprint, I have a simple js idea to open a ticket about.

**Comment by josephgknox** [comment]
Woot!

**Pull request merged**